### PR TITLE
feat: add paginated credentials, delete credential endpoinst, PATCH method to CORS

### PR DESCRIPTION
**Pull request by volodymyr-basiuk**
- added GET /credentials/search endpoint with FTS and pagination
- added DELETE /credentials/{id} endpoint
- added PATCH AllowedMethod to CORS settings
- extend GET /identities with `authBJJCredentialStatus`, `method`, `blockchain` and `network`

### CODE REVIEW

File: api/api.yaml [hunk 1]
```diff
@@ -635,6 +635,83 @@ paths:
...
+            type: string
+            example: did:polygonid:polygon:amoy:2qFpPHotk6oyaX1fcrpQFT4BMnmg8YszUwxYtaoGoe
+        - in: query
+          name: status
+          schema:
+            type: string
+            enum: [ all, revoked, expired ]
+          description: >
+            Schema type:
+              * `all` - All Schemas. (default value)
```
Comment (martinsaporiti) [inline api/api.yaml:671]: this should be credentials?

**Review by martinsaporiti — CHANGES_REQUESTED** [review]
Could you add some tests?

**Comment by volodymyr-basiuk** [comment]
> Could you add some tests?

added

**Review by martinsaporiti — APPROVED** [review]
LGTM

**Review by martinsaporiti — APPROVED** [review]
LGMT
good work

**Pull request merged**